src/pages/EmployeePage/hooks/useGetEmployee.ts

```diff
@@ -0,0 +1,34 @@
+import { bindActionCreators } from 'redux'
+
+import { useLazyQuery } from '@apollo/client'
+
+import { EMPLOYEE_QUERY } from '@/GraphQL/queries'
+import { actionCreators, useAppDispatch } from '@/state'
+
+const useGetEmployee = () => {
```

Comment:
why this logic? why not just a hook that requests emp and returns him. Why do you need this complicate callback